config/install/core.entity_view_display.media.video.default.yml

```diff
@@ -0,0 +1,60 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.video.field_id
+    - field.field.media.video.field_source
+    - field.field.media.video.field_video
+    - media_entity.bundle.video
+  module:
+    - user
+    - video_embed_field
+_core:
+  default_config_hash: 2estpN3n0MPLAQk-7-pHJljmIvjS5nB2thalsLpl_sM
+id: media.video.default
+targetEntityType: media
+bundle: video
+mode: default
+content:
+  created:
+    label: hidden
+    type: timestamp
+    weight: 1
+    settings:
+      date_format: medium
+      custom_date_format: ''
+      timezone: ''
+    third_party_settings: {  }
+  field_source:
+    type: string
+    weight: 4
+    label: above
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+  field_video:
+    type: video_embed_field_video
+    weight: 3
+    label: hidden
+    settings:
+      responsive: true
+      width: 854
+      height: 480
+      autoplay: true
+    third_party_settings: {  }
+  name:
+    label: hidden
+    type: string
+    weight: 0
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+  uid:
+    label: hidden
+    type: author
+    weight: 2
+    settings: {  }
+    third_party_settings: {  }
+hidden:
+  field_id: true
+  thumbnail: true
```

Comment:
@slashrsm the field_id doesn't look nice there. 

Below the video you can see the id of the video. So in case of a youtube video ex: https://www.youtube.com/watch?v=mpbWQbkl8_g the id would be mpbWQbkl8_g and looks weird.